source/Octopus.Tentacle.Tests.Integration/Support/TentacleBuilder.cs

```diff
@@ -297,6 +265,40 @@ public ITentacleBuilder InstallAsAService()
             return (runningTentacle, serviceUri);
         }
 
+        async Task SetEnvironmentVariablesForService(string instanceName, TemporaryDirectory tempDirectory, ILogger logger, CancellationToken cancellationToken)
```

Comment:
I like that this sets it for the service only (so it won't affect other tests).

It would be nice to not have to poke the registry. But at least it should clean up after itself when we uninstall the service.

An alternative could be to provide a command line argument to Tentacle to allow it to set variables for itself (maybe?). But a bridge too far for fixing tests 🙂